ConfigSpace/hyperparameters.pyx

```diff
@@ -2556,7 +2556,14 @@ cdef class CategoricalHyperparameter(Hyperparameter):
             Probability density values of the input vector
         """
         probs = np.array(self.probabilities)
+        vector = np.atleast_1d(vector)
```

Comment:
Hey, is there a reason to add this only for categorical hyperparameters?

Reply:
Not sure why I added this, but everything is still working without this line. I removed it.